app/models.py

```diff
@@ -76,7 +78,10 @@ def update_from_original(self, original):
                 current_app.logger.debug("{} has no column {} to copy from".format(original, c.name))
 
 
-class User(db.Model):
+BaseModel: DefaultMeta = db.Model
```

Comment:
This file gave me some trouble. `db.Model` is a type rather than a variable (or something like that?) so MyPy complains about using it as a base class, even though this seems to be the recommended way to use flask-sqlalchemy: https://flask-sqlalchemy.palletsprojects.com/en/2.x/models/

I tried the work around suggested here: dropbox/sqlalchemy-stubs#76 (comment) and it worked!